src/go/rpk/pkg/cli/cmd/plugin/plugin.go

```diff
@@ -294,3 +320,47 @@ func determineBinDir() (string, error) {
 	}
 	return paths[0], nil
 }
+
+func tryDirectDownload(name, version string) ([]byte, error) {
+	u := fmt.Sprintf("https://dl.redpanda.com/rpk-plugins/raw/names/%s-%s-%s/versions/%s/%[1]s.tgz", name, runtime.GOOS, runtime.GOARCH, version)
+	fmt.Printf("Searching for plugin %q in %s...\n", name, u)
+
+	client := &http.Client{Timeout: 30 * time.Second}
```

Comment:
I prefer having a timeout instead of hanging and making the user hit ctrl+c. The user doesn't know how big is the plugin they are trying to download (8 - 10Mb right now).

I'm going to increase the timeout to 100, a slow internet (~1 Mbps) can take about 80s to download our plugins.